cmd/brimcap/ztests/analyze-process-error.yaml

```diff
@@ -0,0 +1,38 @@
+script: |
+  export PATH=$(pwd):$PATH
```

Comment:
Maybe say `cmd: ./errorproc.sh` in the configuration files so you don't need this?

Reply:
Does not work, im thinking because cwd for the process is a tempdir?